roboteam_ai/src/control/ballHandling/DribbleBackwards.h

```diff
@@ -0,0 +1,86 @@
+//
+// Created by thijs on 25-5-19.
+//
+
+#ifndef ROBOTEAM_AI_DRIBBLEBACKWARDS_H
+#define ROBOTEAM_AI_DRIBBLEBACKWARDS_H
+
+#include <roboteam_utils/Angle.h>
+#include <roboteam_utils/Vector2.h>
+#include "roboteam_ai/src/control/positionControllers/RobotCommand.h"
+#include "roboteam_ai/src/world/Robot.h"
+
+namespace rtt {
+namespace ai {
+namespace control {
+
+class RotateAroundBall;
+class RotateAroundRobot;
+class DribbleBackwards {
+    public:
+        enum BackwardsProgress : short {
+          START,
+          TURNING,
+          APPROACHING,
+          OVERSHOOTING,
+          DRIBBLING,
+          DRIBBLE_BACKWARDS,
+          SUCCESS,
+          FAIL
+        };
+        BackwardsProgress getBackwardsProgression();
+
+    private:
+        RotateAroundBall* rotateAroundBall;
+        RotateAroundRobot* rotateAroundRobot;
```

Comment:
Again naming is confusing. I like rotateWithBall more

Reply:
fixed